charts/agent-control/tests/install_flux_test.yaml

```diff
@@ -3,9 +3,10 @@ tests:
   - it: should leverage correct image tag
     set:
       agent-control-cd:
-        installerImage:
-          tag: 123
-          repository: test
+        installer:
+          image:
+            tag: 123
+            repository: test
```

Comment:
I'm not sure if it is worth it: we could extend the uni-tests with the recently introduced `extraVolumes` and `extraEnv`